emcc.py

```diff
@@ -2732,7 +2732,7 @@ def do_binaryen(target, options, wasm_target):
       final_js = building.minify_wasm_js(js_file=final_js,
                                          wasm_file=wasm_target,
                                          expensive_optimizations=will_metadce(),
-                                         minify_whitespace=minify_whitespace(),
```

Comment:
I guess this because if we minify whitespace here it would be:

a) redundant
b) make closure error message less useful

Maybe worth a comment?